### ISSUE TITLE
Vidéos not showing 

### ISSUE THREAD

**Issue by jarod5001** [photoview/photoview#370]
Installed photoview on yunohost server. 
Videos are not working, I'm getting this error on scan requests :

Failed to begin database transaction: failed to process photo (/home/yunohost.multimedia/user/Picture/test/mp4.mp4): failed to read metadata for original video (mp4.mp4): read video stream metadata: could not read video metadata (mp4.mp4): error running ffprobe [] fork/exec /usr/bin/ffprobe: resource temporarily unavailable

And when clicking on the video I get this error message :
could not get media by media_id and user_id from database: record not found

Any recommend compression format? Is there a way to reset the database?
Thanks

**Comment by viktorstrate** [comment]
Is the file `/home/yunohost.multimedia/user/Picture/test/mp4.mp4` a valid mp4 file?

If it is valid, do you then get the same error if you restart the Docker container and rescan?

**Comment by jarod5001** [comment]
Yes it's a valid mp4, it's a reencoded video using videocoder, it plays correctly on pc and phone.
I'm not using docker. It's a yunohost server
https://yunohost.org/en?q=/en/
I reinstalled photoview and launched the scan (about 6k photos and videos) and will check it later.
Thanks

**Comment by viktorstrate** [comment]
Okay another reason might be that Photoview spawns more child-processes than the operating system / virtualization environment allows. To mitigate against this, you could try to reduce the max concurrent workers in the Settings page.

**Comment by jarod5001** [comment]
That's great, it worked.

**Comment by fmnamado** [comment]
Hello,
I am also having this problem. I am running photo view this in docker, rpi4, raspberry pi os 11.
In Portainer I can see some info.

In logs:

`2022/03/30 08:17:38 ERROR: Failed to begin database transaction: failed to process photo (/path/to/video/IMG_4827.MOV): could not read video metadata (IMG_4827.MOV): error running ffprobe [] signal: killed`

In stats, I can see some additional info on the processes, spawning and killing:
`ffprobe -loglevel fatal -print_format json -show_format -show_streams /path/to/video/IMG_4827.MOV`

All videos (99%) are taken from iPhone 5s, 6 or XR. A lot of videos get scanned and transcoded with no trouble, but I am having this in a lot of videos also.
Some h264 (the past codec), the more recent ones are HEVC. Both kind of videos were correctly scanned in photoview in other files.

I would like some help to troubleshoot and/or understand the pattern.
I already disabled the max processes to search to only one (it was the first thing I made when commissioning the software).

Can you suggest any kind for tests either in photo view or in the videos themselves?
I would say the problem is NOT in the videos, but just in case, I would double check.

Thank you!

**Comment by fmnamado** [comment]
Ups I didn't see it was closed thread. Can we proceed on this thread?

**Comment by kkovaletp** [comment]
I'm closing this issue again, as it looks to be a configuration/compatibility issue for the specific environment, and it was reported for an older version.
Now we have many changes done and it should work fine on the supported systems.
